### PR TITLE
Create suppress key for xslt streaming warning

### LINKED ISSUE #2775: No option to suppress warning "ForEachChildElementPipe [] XsltProcessor xsltVersion [2] currently does not support streaming XSLT, might lead to memory problems for large message"
**❗ please do not add sensitive information in issues, you can provide extra information via email using issue number as reference ❗**

**Describe the issue**
There is no option to suppress the warning "ForEachChildElementPipe [] XsltProcessor xsltVersion [2] currently does not support streaming XSLT, might lead to memory problems for large messages".

Usually XSLT v1 has enough functionality for elementXPathExpression, but XPath 2 has more functions and sometimes those are needed to filter certain elements out correctly. For small XMLs it is then fine to set xsltVersion 2, but there is no option to suppress the warning.

**Reporter**
Ricardo

**To Reproduce**

**Screenshots**

**Configuration**
```
Please provide the configuration of the Pipe or Receiver with the problem.
```

**Input**
```
Please provide an example of the input message. 
Alternatively, provide a Ladybug report.
```

**Environment**
```
Please go to the console of your Frank, click on 'Information' and copy-paste the results here. 
```

**Additional Environment**
 - DBMS: [e.g. Oracle, MSSQL, MariaDB, MySql, PostgreSQL]
 - Browser: [e.g. Chrome, Safari, Edge, Firefox]

**Additional Context**
Add any other context about the problem here. (f.e. ladybug report / test adapter with larva test)

### PR DESCRIPTION
**Pull request by alisihab**
Closes #2775

### CODE REVIEW

File: core/src/main/resources/AppConstants.properties [hunk 1]
```diff
@@ -180,6 +180,10 @@ xml.namespaceAware.default=false
 xslt.streaming.default=true
 xslt.auto.reload=false
 xslt.bufsize=4096
+# ForEachChildElement pipe usually works with large files and uses streaming.
+# Currently streaming is only supported with xslt version 1.
+# Therefore this property will not change the xslt version in ForEachChildElementPipe.
+# If desired, xsltVersion can be set for the individual pipe.
```
Comment (gvanbrakel) [inline core/src/main/resources/AppConstants.properties:186]: This is a confusing comment if you put is this way. Please first explain/document the property as it is, and then note the exception

**Review by gvanbrakel — CHANGES_REQUESTED** [review]

File: core/src/main/resources/AppConstants.properties (hunk 1 above)
Comment (nielsm5) [inline core/src/main/resources/AppConstants.properties:186]:
```suggestion
# Sets the default XSLT version throughout the entire application except the ForEachChildElementPipe.
# The ForEachChildElementPipe usually works with large files and thus is streaming preferred.
# Currently streaming is only supported with XSLT version 1.
# Therefore this property will not change the XSLT version in the ForEachChildElementPipe.
# If desired, the XSLT version can be change for each individual ForEachChildElementPipe using xsltVersion=2.
```

Comment (nielsm5) [inline core/src/main/resources/AppConstants.properties:186]:
```suggestion
# Sets the default XSLT version throughout the entire application except the ForEachChildElementPipe.
# The ForEachChildElementPipe usually works with large files and thus is streaming preferred.
# Currently streaming is only supported with XSLT version 1.
# Therefore this property will not change the XSLT version in the ForEachChildElementPipe.
# If desired, the XSLT version can be change for each individual ForEachChildElementPipe using xsltVersion=2.
```

**Comment by gvanbrakel** [comment]
Could you provide a backport to 7.7 too?

**Review by gvanbrakel — APPROVED** [review]

**Review by nielsm5 — APPROVED** [review]

**Pull request merged**